R/shiny.R

```diff
@@ -169,7 +196,8 @@ app_server <- function(input, output, session) {
     observe({
         shinyWidgets::updateVirtualSelect(
             "genes",
-            choices = shinyWidgets::prepare_choices(promoters, label, coords)
+            choices = shinyWidgets::prepare_choices(promoters, label, coords),
+            selected = "chrX:60843429-60943430"
```

Comment:
remove the default selected - might not work with different assemblies. We can add the default to run_app as a parameter.